### PR TITLE
Nextcom

### PR DESCRIPTION
**Pull request by ebarland**
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

**PS:**
**Unfortunately I was not able to add a label, as the label controls would not show up, but it should be labeled "certified-connector"**. - ebarland

### CODE REVIEW

**Comment by ebarland** [comment]
Unfortunately I was not able to add a label, but it should be labeled "certified-connector".

**Comment by ptalrejagithub** [comment]
Thank you for the submission! This will be reviewed, and all the communication will be here on GitHub. We will get back to you within two business days.

File: certified-connectors/Nextcom/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,585 @@
...
+                  "type": "string",
+                  "description": "Relays the contact ID and, wether a new contact was created or if a duplicate was found and re-used.",
+                  "title": "Insert status"
+                }
+              }
+            }
+          },
+          "400": {
+            "description": "default",
+            "schema": {}
```
Comment (sriyen-msft) [inline certified-connectors/Nextcom/apiDefinition.swagger.json:47]: @ebarland Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide valid response schema definition for all the operations in the swagger.

File: certified-connectors/Nextcom/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,585 @@
...
+                  "title": "Force from list ID",
+                  "items": {
+                    "type": "integer",
+                    "format": "int32"
+                  },
+                  "description": "Used in conjunction with duplicateSettings. If DuplicateSetting 4 is chosen, duplicates will only be moved from the lists supplied in this parameter"
+                },
+                "orgNoOrSSN": {
+                  "type": "string",
+                  "description": "Org number for companies, social security number for private persons",
```
Comment (sriyen-msft) [inline certified-connectors/Nextcom/apiDefinition.swagger.json:125]: @natalie-pienkowska and @ptalrejagithub, Is it safe to take input of SSN and similar highly confidential information from users thru Power Automate. Is it required to let our legal team know about this?

File: certified-connectors/Nextcom/apiProperties.json [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+          "x-ms-apimTemplateParameter.urlTemplate": "https://@connectionParameters('domain').nextcom.no/rest-api-dev/public/v2.0/",
+          "x-ms-apimTemplate-operationName": [
+            "NX_CONTACT",
+            "NX_PIPELINE"
+          ]
+        }
+      }
+    ],
+    "publisher": "Nextcom Evolution AS",
+	"stackOwner": "Nextcom Evolution AS"
```
Comment (sriyen-msft) [inline certified-connectors/Nextcom/apiProperties.json:60]: @ebarland Please remove extra space.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: certified-connectors/Nextcom/apiDefinition.swagger.json (hunk 1 above)
Comment (ebarland) [reply certified-connectors/Nextcom/apiDefinition.swagger.json]: added response schemas for all outputs: 
https://github.com/microsoft/PowerPlatformConnectors/pull/1123/commits/2a4fad1485e9a0b2f9d5fb2494a3f37236b3d607

This did give me warnings in the CLI validation tool though: 

> Swagger certification succeeded with warnings. It is not required but recommended to fix following warnings: 
> Warning : paths/~1crm-system~1contacts/post/responses : The operation 'NX_CONTACT' has more than one response with specified schema, only one schema will be used in the designer.
> Warning : paths/~1crm-system~1pipelines~1contracts/post/responses : The operation 'NX_PIPELINE' has more than one response with specified schema, only one schema will be used in the designer.

File: certified-connectors/Nextcom/apiProperties.json (hunk 3 above)
Comment (ebarland) [reply certified-connectors/Nextcom/apiProperties.json:60]: Fixed spacing: 
https://github.com/microsoft/PowerPlatformConnectors/pull/1123/commits/0bda7a27cd02e427b4d8032e7670ba29ed8e045a

File: certified-connectors/Nextcom/apiDefinition.swagger.json [hunk 4]
```diff
@@ -0,0 +1,597 @@
...
+                  "description": "Relays the contact ID and, wether a new contact was created or if a duplicate was found and re-used.",
+                  "title": "Insert status"
+                }
+              }
+            }
+          },
+          "400": {
+            "description": "Returns error if the input model is malformed",
+            "schema": {
+				"title": "Bad request error",
```
Comment (sriyen-msft) [inline certified-connectors/Nextcom/apiDefinition.swagger.json:48]: @ebarland Indentation - The formatting of both the swagger and apiProperties.json files is not correct. The properties are not aligned properly and there is a longer tab index between the properties. Please refer the certified connectors in the [open source repo](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/certified-connectors/Africa%27s%20Talking%20Airtime/apiDefinition.swagger.json) for reference.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: certified-connectors/Nextcom/apiDefinition.swagger.json (hunk 2 above)
Comment (schabungbam) [reply certified-connectors/Nextcom/apiDefinition.swagger.json:133]: Please make it secure string at least

File: certified-connectors/Nextcom/apiDefinition.swagger.json (hunk 4 above)
Comment (ebarland) [reply certified-connectors/Nextcom/apiDefinition.swagger.json]: Fixed: https://github.com/microsoft/PowerPlatformConnectors/pull/1123/commits/dd2b68272bc676d9fe8d785009643b31e36a5b12

File: certified-connectors/Nextcom/apiDefinition.swagger.json (hunk 2 above)
Comment (ebarland) [reply certified-connectors/Nextcom/apiDefinition.swagger.json:134]: securestring is not a valid swagger 2.0 type.
I added a password-format, tag, as it is the closest I could think of. 
https://github.com/microsoft/PowerPlatformConnectors/pull/1123/commits/d290958dc36d64f2234cb310c1b59fcafed9bc98

**Review by sriyen-msft — APPROVED** [review]

**Pull request merged**